src/lib/PreviewUI.js

```diff
@@ -309,6 +324,10 @@ class PreviewUI {
      * @return {void}
      */
     hideNotification() {
+        if (!this.notification) {
+            this.setupNotification();
+        }
```

Comment:
I don't think you need to setup notification here since it's only a prerequisite to showing a notification.

Something like should work:
```
if (this.notification) {
    this.notification.hide();
}
```